app/views/api/metrics/index.html.slim

```diff
@@ -82,7 +82,7 @@ section.Section.u-toggleableBySettingsBox data-state="open"
 section.Section.u-toggleableBySettingsBox data-state="open"
   h2 Metrics
   p
-    ' Hits are the built-in top-level metric and the parent metric of the methods. Other top level metrics can be added here if needed.
+    ' <em>Hits</em> is the built-in metric and is used to track all hits made to your API. Additional top-level metrics can be added here in order to track other, non hit-based usage.
```

Comment:
Not sure if we should remove the part where we talk about  "top level" nature of hits and the fact that the methods are actually children of the hits metric.

Reply:
The top-level/children terminology is IMO a bit confusing when talking about metrics: _Hits_ tracks _all_ received requests and its a unique metric. Hits can be **classified** however, depending on the method/route but the metric is the same, there are no children or non-top-level metrics.

Reply:
How about:

>  <em>Hits</em> is the built-in metric to which all methods report. Additional top-level metrics can be added here in order to track other usage that shouldn't increase the hit count.

Reply:
👌